apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UpdateObjectOptions.cs

```diff
@@ -149,6 +165,27 @@ internal void ModifyRequest(UpdateRequest request, Object obj)
             {
                 request.UserProject = UserProject;
             }
+            if (OverrideUnlockedRetention != null)
+            {
+                request.OverrideUnlockedRetention = OverrideUnlockedRetention;
+            }
+            if (RetentionMode != null || RetainUntilTime != null)
```

Comment:
Actually, I don't think we need these bits in the optios as calling code is able to build the object itself and set there whatever they need.

We only need OverrideUnlockRetention because that's a request field, and its the request what we build internally for the users.